src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/features/SchedulerConfigTest.java

```diff
@@ -37,7 +39,7 @@ public void should_integrate_with_shedlock() throws Exception {
         ArgumentCaptor<LockConfiguration> configCaptor = ArgumentCaptor.forClass(LockConfiguration.class);
 
         // wait for asynchronous run of the scheduled task in background
-        Thread.sleep(2000);
+        Thread.sleep(5000);
```

Comment:
Was there a need for this increase? 5 seconds is quite a lot of waiting.

Reply:
I have got one failure locally (not in Jenkins), this increase made it stable. Multithreaded tests are generally not very stable

Reply:
If we got to 5 seconds of waiting, maybe we should move to some polling-based approach with timeout, i.e. wait only as long as it's necessary?

Reply:
Reverted back to 2000